### ISSUE TITLE
[Bug]: I can't run repeated measures ANOVA between factors

### ISSUE THREAD

**Issue by Daboxs** [jasp-stats/jasp-issues#3052]
### JASP Version

0.19.1

### JASP Module

ANOVA

### What analysis are you seeing the problem on?

One Way Repeated Measures

### What OS are you seeing the problem on?

Windows 11

### Bug Description

I can't run my anova repeated measures when adding my Between Subject Factors. It shows an error message saying: Duplicate participant column added to model terms" 
![error](https://github.com/user-attachments/assets/f71f4b9f-b10d-4c59-bad3-5ea3e510d569)

### Expected Behaviour

It should show the results in between subjects for this statistics.

### Steps to Reproduce

1. Data set importing from a csv file
2. Conducting Repeated Measures ANOVA
3. Create one 3 level factor
4. Entering the between subject factor using a nominal variable
5. Now, I get this error message
6.
7.
![error](https://github.com/user-attachments/assets/4923e63c-f7ed-49ac-898c-1014fe5a74c1)

...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by JohnnyDoorn** [comment]
Hi @Daboxs ,

In RM ANOVA, participant ID (row number) is already added to the model, so adding a between subjects factor that also has these (essentially) row numbers will result in an error message. Does the analysis work if you remove "patient"?

Kind regards
Johnny

**Comment by Daboxs** [comment]
Yes it does, but how can I get the results for the between subject effect as it requires a between subject factor with not just the Residuals? 
![error2](https://github.com/user-attachments/assets/445e81ad-6f5c-46af-994c-98496a15005c)

**Comment by tomtomme** [comment]
You would need a variable that devides the patients into Independent groups
